client/boot/index.js

```diff
@@ -1,5 +1,9 @@
 // Initialize localStorage polyfill before any dependencies are loaded
 require( 'lib/local-storage' )();
+
+// Initialize draft-js polyfill needed for SEO title format editor
+require( 'lib/draft-js-polyfill' )();
```

Comment:
@mtias From what I've gathered, the draft-js is already part of the same chunk as SEO. Furthermore, the polyfill was already placed at the start of that chunk. It seems like the issue was caused by importing `parser.js`, that in turn imported `draft-js`, before the polyfills were defined.

> The main issue is that loading at the start of boot doesn't take into account that most sections of Calypso don't need to run draft-js at the moment.

I agree, that was my concern with original approach too. I have completely revamped the previous solution in order to remove those polyfills from boot.